varnishgather

```diff
@@ -510,6 +510,8 @@ mycat /etc/init.d/vac
 mycat /opt/vac/etc/defaults
 mycat /var/opt/vac/log/vac-stderr.log
 mycat /var/opt/vac/log/vac.log
+mycat /usr/lib/systemd/system/vac.service
+mycat /etc/systemd/system/vac.service.d/override.conf
```

Comment:
I think this will be a bit error prone since the override files can have other names than ``override.conf``. There is a tool called ``systemd-delta`` that shows overrides, but I don't see how to execute it for specific services.

Reply:
You can get the paths from `systemctl show --value -pDropInPaths,FragmentPath vac.service` and then just loop over those.

Reply:
Nice! Thanks for the pointer. It seems ``--value`` is not supported in systemd on EL7 though. It is version 219, probably with a lot of patches on top (release 57).

Reply:
That's easy enough to replace with a `| cut -d= -f 2-`.

Reply:
for file in /etc/systemd/system/vac.service.d/*; do
    mycat $file"
done